packages/lib/src/components/tx-variants.ts

```diff
@@ -143,6 +143,7 @@ export enum TxVariants {
     upi = 'upi', // also QR
     upi_qr = 'upi_qr', // also QR
     upi_collect = 'upi_collect', // also QR
+    upi_intent = 'upi_intent', // also QR
```

Comment:
I believe the `upi_intent` is missing in the `components-map.ts` and `components-name-map.ts` , no?

Reply:
Indeed nice find, will add.